### PR TITLE
Use langtool to insert missing strings into language inis

### LINKED ISSUE #14905: [Translation] Report on "ja_JP.ini".
### What should happen

I cant find some keywords in ja_JP.ini.

In "Control" section.

Eng)　Hide touch analog stick background circle
Jap )　タッチ用アナログスティックの外周円を非表示にする

Eng)　Gesture mapping
Jap)　ジェスチャー操作の設定をする

Eng)　Gesture
Jap)　ジェスチャー

Eng)　Enable gesture control
Jap)　ジェスチャー操作を有効にする

Eng)　Swipe
Jap)　スワイプ

Eng)　Swipe Up
Jap)　スワイプ【上】

Eng)　Swipe Down
Jap)　スワイプ【下】

Eng)　Swipe Left
Jap)　スワイプ【左】

Eng)　Swipe Right
Jap)　スワイプ【右】

Eng)　Swipe sensitivity
Jap)　スワイプの感度

Eng)　Swipe smoothing
Jap)　スワイプの滑らかさ

Eng)　Double tap
Jap)　ダブルタップ

Eng)　Double tap button
Jap)　ダブルタップ時の設定

### Who would this benefit

All Japanese user.

### Platform (if relevant)

Android

### Games this would be useful in

All gesture controlable games.

### Other emulators or software with a similar feature

_No response_

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

### PR DESCRIPTION
**Pull request by hrydgard**
Makes a few fixes to langtool too.

Fixes #14905  (at least mostly)

### CODE REVIEW

File: assets/lang/es_ES.ini [hunk 1]
```diff
@@ -605,23 +649,41 @@ An.Right = Stick derecha
 An.Up = Stick arriba
 Analog limiter = Limitador analóg.
 Analog Stick = Stick analógico
+Arrow down = ลูกศรลง
```
Comment (iota97) [inline assets/lang/es_ES.ini:652]: Thai striked again? It seem to be like this in all the .ini (didn't check them all tho') :)

File: assets/lang/es_LA.ini [hunk 2]
```diff
@@ -586,6 +629,7 @@ SavesAreTemporary = PPSSPP está guardando en almacenamiento temporal
 SavesAreTemporaryGuidance = Extraer PPSSPP en cualquier parte para guardar para siempre
 SavesAreTemporaryIgnore = Ignorar advertencias
 UnpinPath = Desanclar
+UseBrowseOrLoad = UseBrowseOrLoad
```
Comment (iota97) [inline assets/lang/es_LA.ini:632]: Using key rather than default string

File: assets/lang/es_LA.ini [hunk 3]
```diff
@@ -790,6 +870,7 @@ Scanlines = Líneas de escáneo (CRT)
 Sharpen = Afilado
 SSAA(Gauss) = Supersampleado AA (Gauss)
 Tex4xBRZ = 4xBRZ
+TexMMPX = TexMMPX
```
Comment (iota97) [inline assets/lang/es_LA.ini:873]: Using key rather than default string

File: assets/lang/es_LA.ini [hunk 4]
```diff
@@ -856,6 +937,8 @@ Stopping.. = Deteniendo...
 
 [Reporting]
 Bad = Mal
+FeedbackCRCCalculating = FeedbackCRCCalculating
+FeedbackCRCValue = FeedbackCRCValue
```
Comment (iota97) [inline assets/lang/es_LA.ini:941]: Using key rather than default string

**Comment by hrydgard** [comment]
I think this should be OK now, and if there are still mistakes, they're pretty easy to fix thanks to the tool (and good ol' search and replace if needed).

File: assets/lang/th_TH.ini [hunk 5]
```diff
@@ -997,8 +997,8 @@ GLToolsWarning = ระวัง: ตรวจพบ GLTools อาจก่อ
 In menu = ในเมนู
 Load savestate failed = ล้มเหลวในการโหลดเซฟสเตท
 Loaded State = โหลดสเตทเกมเรียบร้อย
-Loaded. Game may refuse to save over different savedata. = โหลดสเตทเรียบร้อยแล้ว ระบบของเกมอาจจะสับสน เมื่อบันทึกด้วยข้อมูลที่แตกต่างจากเซฟธรรมดา
-Loaded. Game may refuse to save over newer savedata. = โหลดสเตทเรียบร้อยแล้ว ระบบของเกมอาจจะสับสน เมื่อบันทึกด้วยข้อมูลที่ใหม่กว่าเซฟธรรมดา
+Loaded. Game may refuse to save over different savedata = Loaded. Game may refuse to save over different savedata
+Loaded. Game may refuse to save over newer savedata = Loaded. Game may refuse to save over newer savedata
```
Comment (Saramagrean) [inline assets/lang/th_TH.ini:1001]: change to english again. 😅

Comment (hrydgard) [reply assets/lang/th_TH.ini:1001]: Haha oops. I'll fix it up

**Pull request merged**